control-plane/csi-driver/src/bin/node/main_.rs

```diff
@@ -327,6 +337,35 @@ pub(super) async fn main() -> anyhow::Result<()> {
         check_ana_and_label_node(&kube_client, node_name, nvme_enabled).await?;
     }
 
+    let conn_fallback = matches.get_flag("nvme-connect-fallback");
+    // Check and store if this initiator node has an RDMA device so that nvme connect over RDMA
+    // can be done. If this node is also an io-engine node with rdma enabled and working,
+    // then this check will always set capability true.
+    let ibv_output = tokio::process::Command::new("ibv_devinfo")
+        .arg("-l")
+        .output()
+        .await;
+
+    let cap = match ibv_output {
+        Ok(okstdout) if okstdout.status.success() => {
+            // In an unlikely case string from utf8 fails, err towards incapability.
+            let outstr = String::from_utf8(okstdout.stdout).unwrap_or("0 HCA".to_string());
+            if !outstr.starts_with("0 HCA") {
+                info!("host node rdma capable. conn_fallback({conn_fallback:?}), {outstr:?}");
+                // todo: check for the presence of nvme_rdma too. But don't bail out as we do for
+                // nvme_tcp.
+                true
+            } else {
+                false
+            }
+        }
+        Ok(_) | Err(_) => {
+            error!("Error executing ibv_devinfo, or invalid command output. conn_fallback({conn_fallback:?}), {ibv_output:?}");
+            false
+        }
+    };
+    let _ = RDMA_CONNECT_CHECK.set((conn_fallback, cap));
```

Comment:
nit: I suggest you not use once cell, and simply store this in the service

Reply:
As discussed, keeping it as-is.